### PR TITLE
Ensure Genome_build is String in Input

### PR DESCRIPTION
**Pull request by mikaell**
This PR adds a functionality or fixes a bug.

`genome_build` is inconsistently a string or an integer through the system. This PR ensures `genome_guild` is cast to string on input from config file and CLI. It also introduces two constants representing version.

* Constants are not yet used throughout the code base
* Casting to string is still necessary as `genome_build` is an integer in most (all?) database entries.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/build/case.py [hunk 1]
```diff
@@ -193,10 +193,9 @@ def build_case(case_data, adapter):
     case_obj["dynamic_gene_list"] = []
 
     # Meta data
-    genome_build = case_data.get("genome_build", "37")
-    if not genome_build in ["37", "38"]:
-        pass
-        ##TODO raise exception if invalid genome build was used
+    genome_build = case_data.get("genome_build", GENOME_BUILD_37)
+    if not genome_build in [GENOME_BUILD_37, GENOME_BUILD_38]:
```
Comment (northwestwitch) [inline scout/build/case.py:197]: Yes I think this is the way to go! 👍🏻

File: scout/constants/__init__.py [hunk 2]
```diff
@@ -95,8 +95,10 @@
     "variant",
     "acmg",
 ]
+GENOME_BUILD_37 = "37"
+GENOME_BUILD_38 = "38"
+BUILDS = [GENOME_BUILD_37, GENOME_BUILD_38, "GRCh38"]
```
Comment (northwestwitch) [inline scout/constants/__init__.py:100]: Who knows why here we had also GRCh38.. Perhaps it's needed to visualize the alignments?

**Comment by mikaell** [comment]
Shall we move forward with this PR or drop it? @dnil @northwestwitch 

It's a small change. And I think it would improve the code a tiny little bit. On the other hand it's not critical.

File: scout/constants/__init__.py (hunk 2 above)
Comment (dnil) [reply scout/constants/__init__.py:100]: No idea, but the RNA view seems like the place to start looking?

**Comment by dnil** [comment]
> Shall we move forward with this PR or drop it? @dnil @northwestwitch
> 
> It's a small change. And I think it would improve the code a tiny little bit. On the other hand it's not critical.

It has caused issues and confusion in the past, so making it crystal clear would not be bad. Just don't spend a lot of effort on it unless you feel you really should do it; there are issues open that really would help the clinicians as well.

As for how to solve it, I don't have a strong opinion: both keeping naming open, as long as the users (other centers) are careful and understand it, or locking it, as any kind of advanced users would be able to just fork and edit as needed.

In the ideal world we would have build handling, where the users could add new builds dynamically, perhaps with aliases for their names and so on. But not mix them up. 

Be mindful of what is already stored on the db, and what is available to the running code in addition to the CLI.

**Comment by mikaell** [comment]
Closing this food-for-thought draft.

**Pull request closed without merging**